synapse/lib/cell.py

```diff
@@ -2623,17 +2623,7 @@ def walkpath(path):
         walkpath(self.backdirn)
         return backups
 
-    async def iterBackupArchive(self, name, user):
-
-        success = False
-        loglevel = logging.WARNING
-
-        path = self._reqBackDirn(name)
-        cellguid = os.path.join(path, 'cell.guid')
-        if not os.path.isfile(cellguid):
-            mesg = 'Specified backup path has no cell.guid file.'
-            raise s_exc.BadArg(mesg=mesg, arg='path', valu=path)
-
+    async def _streamBackupArchive(self, path, user, name):
```

Comment:
Tidy thing while w'ere here - we should have a hard assertion that we have a link here from the scope, since this is called from top level Cell APIs which are not private but implied to be coming from a CellApi subclass.

Reply:
makes sense, will update for next commit.